### ISSUE TITLE
When generating XmlArrayItemAttribute namespace is missing?

### ISSUE THREAD

**Issue by AnderssonPeter** [mganss/XmlSchemaClassGenerator#253]
Hi when `XmlSchemaClassGenerator` generates `XmlArrayItemAttribute` it dosen't generate the namespace and if the type its pointing to is in a different namespace it generates a compilation failure.

Example:
```
[XmlArrayItemAttribute("AbstractGeometricPrimitive", Type=typeof(AbstractGeometricPrimitiveType), Namespace="http://www.opengis.net/gml/3.2")]
``` 
Type from same namespace that works!
```
public GML.v1_1.GeometryPropertyType Geometri { get; set; }
```

**Comment by mganss** [comment]
Can you provide an example schema?

**Comment by AnderssonPeter** [comment]
If i run the following `xscgen http://namespace.lantmateriet.se/distribution/produkter/byggnad/v2/byggnad-2.0.3.xsd --csm=Public --ct=System.Array --nullable -noUnderscore --compactTypeNames --verbose`
Then open `Distribution.Produkter.Byggnad.V2.cs` and look at line 1137 - 1146.

**Comment by mganss** [comment]
Thanks. Fixed in 2.0.512.
